cif_ed.dic

```diff
@@ -0,0 +1,643 @@
+    _type.purpose                 State
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Text
+
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         rm
+;
+         'reference molecule' Absolute configuration established by the
+         structure determination of a compound containing a chiral reference
+         molecule of known absolute configuration.
+;
+         ad
+;
+         'anomalous dispersion' Absolute configuration established by a-d
+         effects in diffraction measurements on the crystal.
+;
+         dyn
+;
+         Absolute configuration determined directly from dynamical refinement.
+;
+         rmad
+;
+         'rm + ad' Absolute configuration established by the structure
+         determination of a compound containing a chiral reference molecule of
+         known absolute configuration and confirmed by a-d effects in
+         diffraction measurements on the crystal.
+;
+         syn
+;
+         'synthetic' Absolute configuration has not been established by
+         anomalous-dispersion effects in diffraction measurements on the
+         crystal. The enantiomer has been assigned by reference to an
+         unchanging chiral centre in the synthetic procedure.
+;
+         unk
+;
+         'unknown' No firm chemical or a-d evidence for an assignment is
+         available. An arbitrary choice of enantiomer has been made.
+;
+         .
+;
+         Inapplicable.
+;
+
+save_
+
+save_diffrn.precession_angle
+
+    _definition.id                '_diffrn.precession_angle'
+    _alias.definition_id          '_diffrn_precession_angle'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Angle at the sample of tilt from the normal direction of a radiation
+    beam moved in a precessional scan.
+;
+    _name.category_id             diffrn
+    _name.object_id               precession_angle
+    _type.purpose                 Measurand
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            0.0:
+    _units.code                   degrees
+
+save_
+
+save_diffrn.precession_angle_su
+
+    _definition.id                '_diffrn.precession_angle_su'
+    _alias.definition_id          '_diffrn_precession_angle_su'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Standard uncertainty of _diffrn.precession_angle.
+;
+    _name.category_id             diffrn
+    _name.object_id               precession_angle_su
+    _name.linked_item_id          '_diffrn.precession_angle'
+    _type.purpose                 SU
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   degrees
+
+save_
+
+save_diffrn_measurement.method_precession
```

Comment:
Items with the "yes/no" values in the `CIF_CORE` dictionary often have the "_flag" postfix, so I would propose renaming the item to `_diffrn_measurement.precession_method_flag` or `_diffrn_measurement.precession_flag`.

Reply:
Will include as an issue for community consideration.